staging/src/k8s.io/apiserver/pkg/audit/policy/reader.go

```diff
@@ -49,6 +49,10 @@ func LoadPolicyFromFile(filePath string) (*auditinternal.Policy, error) {
 		return nil, err.ToAggregate()
 	}
 
-	glog.V(4).Infof("Loaded %d audit policy rules from file %s\n", len(policy.Rules), filePath)
+	policyCnt := len(policy.Rules)
+	if policyCnt == 0 {
+		return nil, fmt.Errorf("failed to load policy rules from file %s", filePath)
+	}
+	glog.V(4).Infof("Loaded %d audit policy rules from file %s\n", policyCnt, filePath)
```

Comment:
nit: glog messages shouldn't end in a `\n` (there is always an implicit newline).